.gitpod.yml

```diff
@@ -0,0 +1,13 @@
+image:
+  file: .gitpod.Dockerfile
+
+tasks:
+  - init: yes | lamdera reset
+    command: sudo lamdera live
+ports:
+  - port: 8000
+    onOpen: open-preview
+
+vscode:
+  extensions:
+    - elmtooling.elm-ls-vscode
```

Comment:
Is there a way to configure the Elm path to point to the Lamdera binary? That would probably alleviate some issues down the line.

Reply:
I added a .settings file to set this for the elm LS. Its breaking now though and I'm not sure why 😴